Data/src/main/java/org/tribuo/data/columnar/processors/field/DoubleFieldProcessor.java

```diff
@@ -41,17 +41,48 @@
     @Config(mandatory = true,description="The field name to read.")
     private String fieldName;
 
+    @Config(description="Emit a feature using just the field name.")
+    private boolean onlyFieldName;
+
+    @Config(description="Throw NumberFormatException if the value failed to parse.")
+    private boolean throwOnInvalid;
+
     /**
      * For olcut.
      */
     private DoubleFieldProcessor() {}
 
     /**
      * Constructs a field processor which extracts a single double valued feature from the specified field name.
+     * <p>
+     * Generates features named "fieldName@value", and does not throw an exception if the value failed to parse.
```

Comment:
It's not 100% clear on first reading that `fieldName` is the name of the field while `value` is literally the string `value`. Is there a way to make this clearer? Maybe make `fieldName` a reference in the javadoc?

Reply:
Sure.